rwkv/measure_pexplexity.py

```diff
@@ -17,20 +16,30 @@ def parse_args():
     parser.add_argument('model_path', help='Path to model checkpoint file', type=str)
     parser.add_argument('text_path', help='Path to text file in UTF-8 encoding', type=str)
     parser.add_argument('ignore_first_n_tokens', help='How many tokens should be skipped before loss is measured', type=int)
-    parser.add_argument('token_limit', help='How many tokens to process; set to -1 to process all text', nargs='?', type=int, default=-1)
+    parser.add_argument('token_limit', nargs='?', help='How many tokens to process; set to -1 to process all text', type=int, default=-1)
```

Comment:
```suggestion
    parser.add_argument('token_limit', help='How many tokens to process; set to -1 to process all text', nargs='?', type=int, default=-1)
```
`nargs` goes after `help` in all other places.